app/controllers/api/v2/discovery_rules_controller.rb

```diff
@@ -52,7 +52,7 @@ def create
           param :name, String, :required => true
```

Comment:
The whole `discovery_rule` hash could be replaced by `param_group :discovery_rule, :as => :update`.